Add files via upload

ssheminway opened the pull request:
<!--- Thank you for your submission to geoBoundaries! -->
<!--- If you are submitting a boundary, please make sure to include the ISO code and ADM level --> DEU ADM0
<!--- We require full data lineage for all products, but give us your source and we'll do the legwork to hunt down everything we can! --> Public Domain license on Map from Wiki Commons

## Why do we need this boundary?  
<!--- If this is in response to a github issue, just link it here. --> gB Open DEU ADM0 #1823
<!--- Otherwise, let us know why this boundary is better than what's in the current database. -->

## Anything Unusual?
<!--- Please describe any known complications with your submission. -->
<!--- You can also include anything else we need to know while assessing this data. --> Shapefile digitized from a map png

sgmurphy216 reviewed (changes requested):
Hey Selwyn! This submission looks great- just a few little changes and we'll be able to merge.
For 'Boundary Type' - think there's just a little typo here, should be ADM0
*technically when citing geoBoundaries the g isn't capitalized but not a biggie
For 'Release Type', look back at the original issue to see where the file should go. Since this is a gB OPEN issue, we can just include gB OPEN and not include the other releases in this field.
Meta.txt is accidentally named 'meta.txt.txt' - thank you for being thorough, but one .txt will suffice! 
Lastly- no need to add a 'File' field, the bot is stubborn and won't accept the submission if there are extra fields.
Again, a really great submission. Geometries are solid, the digitization looks like you've done this before. Let me know if you have any questions!

ssheminway commented:
Hey Sean, the boundary type was definitely a typo but I changed the release type to just gbOpen and fixed the name to meta.txt. To change the pull request, is there a way to do it within github or do I just upload a different zip file? Also, I'm not sure how the file field got there, is there an easy way to remove it?

sgmurphy216 commented:
> Hey Sean, the boundary type was definitely a typo but I changed the release type to just gbOpen and fixed the name to meta.txt. To change the pull request, is there a way to do it within github or do I just upload a different zip file? Also, I'm not sure how the file field got there, is there an easy way to remove it?

Hey! The file 'field' I was referring to is just a line you have in the meta.txt- that line can just be deleted. As far as updating the PR, navigate in GitHub to the branch that the PR created (like sgmurphy01-patch01), and upload the file to the correct sub folder (gB OPEN) there, the bot should recognize that. 
Let me know if that makes sense- I'll also be in the lab at 5pm today if you have any additional questions!

ssheminway commented:
Oh okay, sorry for the confusion! Just to confirm, I should just delete the
part that is highlighted in the screenshot and not the entire URL?
[image: Screen Shot 2021-09-27 at 12.02.16 PM.png]

On Mon, Sep 27, 2021 at 10:33 AM Sean Murphy ***@***.***>
wrote:

> Hey Sean, the boundary type was definitely a typo but I changed the
> release type to just gbOpen and fixed the name to meta.txt. To change the
> pull request, is there a way to do it within github or do I just upload a
> different zip file? Also, I'm not sure how the file field got there, is
> there an easy way to remove it?
>
> Hey! The file 'field' I was referring to is just a line you have in the
> meta.txt- that line can just be deleted. As far as updating the PR,
> navigate in GitHub to the branch that the PR created (like
> sgmurphy01-patch01), and upload the file to the correct sub folder (gB
> OPEN) there, the bot should recognize that.
> Let me know if that makes sense- I'll also be in the lab at 5pm today if
> you have any additional questions!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/wmgeolab/geoBoundaries/pull/1865#issuecomment-927933975>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVSJZB3Z27B4AMAZ5RN7MULUEB6EFANCNFSM5EZRTMOA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Selwyn Heminway (she/her)
B.S. Geology and Environmental Science
William & Mary '23
***@***.***
